### PR TITLE
fix Estonian language in ng2

### PR DESCRIPTION
**Pull request by kaidohallik**

### CODE REVIEW

File: generators/languages/templates/src/main/webapp/i18n/et/_activate.json [hunk 1]
```diff
@@ -2,7 +2,11 @@
     "activate": {
         "title": "Aktiveerimine",
         "messages": {
+            <%_ if (angularVersion === 'angular1') { _%>
```
Comment (deepu105) [inline generators/languages/templates/src/main/webapp/i18n/et/_activate.json]: the variable has been changed to `clientFw` please update

Comment (kaidohallik) [inline generators/languages/templates/src/main/webapp/i18n/et/_activate.json]: update done

File: generators/client-2/templates/src/main/webapp/app/shared/language/_language.pipe.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ export class FindLanguageFromKeyPipe implements PipeTransform {
             'el': 'Ελληνικά',
             'en': 'English',
             'es': 'Español',
+            'et': 'Eesti',
```
Comment (deepu105) [inline generators/client-2/templates/src/main/webapp/app/shared/language/_language.pipe.ts]: This file has been moved

Comment (kaidohallik) [inline generators/client-2/templates/src/main/webapp/app/shared/language/_language.pipe.ts]: PR updated

**Pull request merged**